demos/HST/run_eureka_wfc3.py

```diff
@@ -9,10 +9,10 @@
 ecf_path = './'
 
 if __name__ == '__main__':
-	s3_meta = s3.reduceJWST(eventlabel, ecf_path=ecf_path)
+	s3_meta = s3.reduce(eventlabel, ecf_path=ecf_path)
 
-	s4_meta = s4.lcJWST(eventlabel, ecf_path=ecf_path, s3_meta=s3_meta)
+	s4_meta = s4.getlc(eventlabel, ecf_path=ecf_path, s3_meta=s3_meta)
```

Comment:
For consistency, maybe we should stick with `genlc`?

Reply:
Or we could change the name of the file - whichever you prefer

Reply:
I'd rather not change the directory structure, docs, etc. Let's use genlc.